src/dmd/root/ctfloat.d

```diff
@@ -230,3 +226,8 @@ extern (C++) struct CTFloat
         half = real_t(0.5);
     }
 }
+
+bool isNaN(real_t r) pure @nogc @safe
```

Comment:
How does this fit with line 44?

Reply:
https://github.com/dlang/dmd/pull/10010#discussion_r291936384